snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/SnomedAxiomSemanticIT.java

```diff
@@ -71,8 +73,9 @@ public void testSnomedAxiomSemantics() throws IOException {
                 StateSet snomedAxiomStatus = Integer.parseInt(columns[2]) == 1 ? StateSet.ACTIVE : StateSet.INACTIVE;
                 EntityProxy.Concept moduleId = EntityProxy.Concept.make(PublicIds.of(UuidUtil.fromSNOMED(columns[3])));
                 EntityProxy.Concept refsetId = EntityProxy.Concept.make(PublicIds.of(UuidUtil.fromSNOMED(columns[4])));
-
-                UUID id = UuidUtil.fromSNOMED(columns[0]);
+//                UUID id = UuidUtil.fromSNOMED(columns[0]);
+//                UUID id = UuidT5Generator.get(UUID.fromString("3094dbd1-60cf-44a6-92e3-0bb32ca4d3de"), columns[0]);
```

Comment:
I would try and use UUID.fromSnomed to see if there is a difference in how UUIDs are being generated, if it works then we know its not a dataset issue, its a UUID generation issue, if they both arent working it may be something to do with the dataset.